runtime/gc_modron_startup/mminit.cpp

```diff
@@ -2587,6 +2587,37 @@ setDefaultConfigOptions(MM_GCExtensions *extensions, bool scavenge, bool concurr
 #endif /* defined(J9VM_GC_LARGE_OBJECT_AREA) */
 }
 
+void
+setConfigOptionsForNoGc(MM_GCExtensions *extensions)
+{
+	/* noScavenger noConcurrentMark noConcurrentSweep, noLOA */
+	extensions->configurationOptions._forceOptionScavenge  = true;
+	extensions->scavengerEnabled = false;
```

Comment:
extensions->scavengerEnabled is declared under "if defined(OMR_GC_MODRON_SCAVENGER)"